### PR TITLE
Style changes - Tech Comms

### PR DESCRIPTION
**Pull request by nic-name**
# Metasys API documentation changes

|Section|Change |
|-----------------------------|--------------------------------------------|
| API version notes| Reworded content under API Notes and configured the note.|
| Version response behaviour| Reworked the content to remove passives and past tenses.|
| Pagination| Reworked the content to remove passives and past tenses.|
| Sorting rules| Change should to must. Removed parentheses, addressed some style guide issues.|
| Relationship links| Reworked the content to remove passives and past tenses.|
| Dates and times| Removed one passive.|
| Validation| Reworked the content to remove passives and past tenses. In the table, I changed the Details column to address some style issues. |

### CODE REVIEW

File: _templates/header.apib [hunk 1]
```diff
@@ -18,70 +18,81 @@ Version: 1
 Prefix: /api/v1
 ```
 
-This documentation was last updated on {{revision}}.
+This documentation was last updated on 26/07/2019.
```
Comment (DMahoney0) [inline _templates/header.apib]: I would suggest we leave this as a variable so its dynamic.

File: _templates/header.apib [hunk 2]
```diff
@@ -18,70 +18,81 @@ Version: 1
...
 
 <!--
 This document describes version 1 of the API (not to be confused with document version in [Document Information](#document-information) section above). -->
 
-While this is currently the only version available, as new versions are added the API consumer will need to specify which version of the API they intend to use. The primary means of doing this is to add the version before the first entity in the URL using the format "v[#]":
+When new versions are added you must specify which version of the API you intend to use. In the future, one release of Metasys, may support multiple versions. In that case you must explicitly specify which version you intend to use in the URL using the format "v[#]":
 
-* `http://localhost/api/v1/spaces` for version 1
-* `http://localhost/api/v2/spaces` for version 2
+For example, `http://localhost/api/v2/spaces` for version 2
```
Comment (DMahoney0) [inline _templates/header.apib]: We had an error in the original examples.  We use HTTPS, so the example would be 'https://localhost/api/v2/spaces'

File: _templates/header.apib [hunk 3]
```diff
@@ -18,70 +18,81 @@ Version: 1
...
+You can also use the following `Accept` header in the HTTP request. To do this, modify the version based on which version you are using in the format v[#].
+For example:
 
 * `Accept: application/vnd.metasysapi.v1+json` for version 1
 * `Accept: application/vnd.metasysapi.v2+json` for version 2
 
 #### Version Specification Rules
 
-Because the version specification is optional and can be provided in multiple ways, the following rules are used when determining or providing version information:
+Version specification is optional. You can configure it in multiple ways. Use the following rules when determining or providing version information:
```
Comment (DMahoney0) [inline _templates/header.apib]: In 10.1 we removed optional versions so this section could be edited or removed.

File: _templates/header.apib (hunk 1 above)
Comment (mgwelch) [reply _templates/header.apib]: Probably a training issue.

File: _templates/header.apib (hunk 2 above)
Comment (mgwelch) [reply _templates/header.apib]: Yeah, looks like this PR used 10.0 as a base. This section is already removed in 10.1. I'm meeting with @nic-name in 5 minutes. We'll discuss.

Comment (mgwelch) [reply _templates/header.apib]: I assume most of the edits will still apply. (And we could actually accept these edits for 10.0 once approved)

**Comment by DMahoney0** [comment]
@michaelgwelch @SheilaBonnar @nic-name I believe this PR is no longer needed.
I wanted to check with you before I closed it.

**Comment by mgwelch** [comment]
> @michaelgwelch @SheilaBonnar @nic-name I believe this PR is no longer needed.
> I wanted to check with you before I closed it.

I believe you are correct as #41 notes that it makes this one (#39) obsolete.

**Pull request closed without merging**